src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4191,9 +4191,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureRefLocalsReturns" xml:space="preserve">
     <value>byref locals and returns</value>
   </data>
-  <data name="IDS_ThrowExpression" xml:space="preserve">
-    <value>&lt;throw&gt;</value>
-  </data>
```

Comment:
Still can't find a good reason that the warning from this duplication doesn't show up on Windows.  Looked around and saw other reports of it. 

Anyways, thanks for removing it.